src/CommonIncludes.js

```diff
@@ -32,6 +32,7 @@ export const DAILY_EVENT_RECORDING_STATS = 'recording-stats';
 export const DAILY_EVENT_RECORDING_ERROR = 'recording-error';
 export const DAILY_EVENT_RECORDING_UPLOAD_COMPLETED =
                'recording-upload-completed';
+export const DAILY_EVENT_RECORDING_DATA = 'recording-data';
```

Comment:
in keeping with the grammar of 'recording-started', 'recording-stopped', etc, I'd go with "recording-got-bytes" or something

Reply:
The parallel I was thinking of is with the 'data' event that MediaRecorder emits. (MediaRecorder is the underlying machinery, here.)